webassets_elm/__init__.py

```diff
@@ -1,11 +1,24 @@
-from os import remove
+from contextlib import contextmanager
+from os import chdir, getcwd, remove
+import os.path
 from sys import platform
 from tempfile import NamedTemporaryFile, TemporaryFile
 from webassets.filter import ExternalTool
```

Comment:
Can we have organized imports as in [PEP8](http://legacy.python.org/dev/peps/pep-0008/#imports)? I tend to put then in alphabetic order inside groups…

Also, let's just `import os` instead of `from os import chdir, getcwd, remove` and `import os.path`… what do you think about it?